tests/tests_LfD/test_gmm_fitting.py

```diff
@@ -9,6 +10,13 @@
 
 
 class ProbabilisticEncodingTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # For some tests, being on ARM or X86 will change the value due floating point
+        # calculations. For example, This difference leads to different GGM even with
+        # no shuffle and the same random seed.
+        cls._is_arm = platform.machine() == "aarch64" or platform.machine() == "armv7l"
```

Comment:
In my understanding you define the architecture of the machine to be able to control those different floating point calculations and be consistent during testing, right?

Reply:
It's even a bit simpler. I don't run the tests that return different values due to floating point optimization on ARM platforms.

Actually, I'm realizing that this is bad design. I'll write a method to know if it's arm instead of saving it as an attribute. It'll make more sense